server/base/src/main/java/org/apache/accumulo/server/master/state/TabletStateChangeIterator.java

```diff
@@ -267,10 +267,6 @@ public static void setMasterState(IteratorSetting cfg, MasterState state) {
 
   public static void setShuttingDown(IteratorSetting cfg, Set<TServerInstance> servers) {
     if (servers != null) {
-      List<String> serverList = new ArrayList<>();
-      for (TServerInstance server : servers) {
-        serverList.add(server.toString());
-      }
       cfg.addOption(SHUTTING_DOWN_OPTION, Joiner.on(",").join(servers));
```

Comment:
I guess this works because join calls toString?

Reply:
I wrote some similar code and it looks like it does